motr/obj.c

```diff
@@ -555,6 +555,30 @@ static int obj_namei_op_init(struct m0_entity *entity,
 	return M0_ERR(rc);
 }
 
+/**
+ * Finds and sets optimal layout id according to the object's size.
+ */
+static void obj_optimal_lid_set(struct m0_obj *obj,
+				struct m0_layout_domain *ldom)
+{
+	uint64_t                *lid;
+
+	lid = &obj->ob_attr.oa_layout_id;
+
+	/* Find optimal layout id when pver id is set and layout id is not */
+	if (obj->ob_attr.oa_layout_id == 0 &&
+		m0_fid_is_set(&obj->ob_attr.oa_pver)) {
+		*lid = m0_layout_find_by_buffsize(ldom, &obj->ob_attr.oa_pver,
+						  obj->ob_attr.oa_obj_size);
+	}
+
+	/* Set default layout id when both layout id and pver id is unset */
+	else if (obj->ob_attr.oa_layout_id <= 0 &&
```

Comment:
Remove this empty line.
```
if (some_condition) {
        some actions;
} else {
        other actions;
}
```

Reply:
1. "obj->ob_attr.oa_layout_id" can be replaced with "*lid";
2. "obj->ob_attr.oa_layout_id" is unsigned int64. So, it would not be less then zero (<0). This condition is the same as "*lid == 0".
3.

Reply:
Done